frontend/components/wrapper/pages/CourseDetail.tsx

```diff
@@ -11,38 +9,19 @@ import {
   upsertFavoriteVideo,
   fetchFavButtonStatus,
 } from '../../../app/api/course/[courseId]/favoriteVideo'
-import { CourseWithSectionsType } from '../../../types/CourseType'
-import { QuestionType } from '../../../types/QuestionType'
 import { useCustomToast } from '../../../hooks/useCustomToast'
-import { AnswerType } from '../../../types/AnswerType'
 import { CourseDetail } from '../../pages/CourseDetail'
 import Error from '../../../app/error'
 import { useDisclosure } from '@chakra-ui/react'
-import { UserProfileType } from '../../../types'
-
-export function CourseDetailWrapper({
-  courseId,
-  initialCourseData,
-  session,
-  userId,
-  videoId,
-  questions,
-  answers,
-  questionId,
-}: {
-  courseId: string
-  session: Session | null
-  initialCourseData: CourseWithSectionsType
-  userId: string
-  videoId: string
-  questions?: QuestionType[]
-  answers: AnswerType[]
-  questionId?: string
-}) {
+import { UserProfileType, WrapperPropsType } from '../../../types'
+
+export function CourseDetailWrapper(WrapperProps: WrapperPropsType) {
```

Comment:
```suggestion
export function CourseDetailWrapper(CourseDetailProps: CourseDetailPropsType) {
```

このコンポーネントへのpropsなので
こういう命名を多くの現場ではしてますね！

Reply:
現場のやり方、勉強になります！
`courseDetailProps` という名前は、別コンポーネント（`frontend/components/pages/CourseDetail.tsx`）で既に使用しているので、`courseDetailWrapperProps` としてみました。